pkg/skaffold/schema/latest/config.go

```diff
@@ -344,6 +344,9 @@ type ClusterDetails struct {
 
 	// RandomDockerConfigSecret adds a random UUID postfix to the default name of the docker secret to facilitate parallel builds, e.g. docker-cfgfd154022-c761-416f-8eb3-cf8258450b85.
 	RandomDockerConfigSecret bool `yaml:"randomDockerConfigSecret,omitempty"`
+
+	// Annotations defines arbitrary non-identifying metadata to a kaniko pod.
```

Comment:
```suggestion
	// Annotations describes the Kubernetes annotations for the pod.
```

And could you please move this nearer the Tolerations so as to keep the Kubernetes related parameters together?

Reply:
Move it right under the `Tolerations` and modify the explanation :) Thank you so much